tests/unit/kvs/kvs_cparams_test.c

```diff
@@ -78,25 +78,6 @@ MTF_DEFINE_UTEST_PRE(kvs_cparams_test, kvs_ext01, test_pre)
     ASSERT_EQ(UINT32_MAX, ps->ps_bounds.as_uscalar.ps_max);
 }
 
-MTF_DEFINE_UTEST_PRE(kvs_cparams_test, suffix_length, test_pre)
```

Comment:
Why remove the test if the rparam still exists?

Reply:
I had removed this as a cparam. But then we decided to add it back as an rparam. So this test needs to go. But I've added back a test for the rparam.